networktype.go

```diff
@@ -0,0 +1,40 @@
+package webrtc
```

Comment:
I created this entire file, which is a mini version of the one found inside of `ice` package because the `ice` package is internal and now that the config can be passed via `SettingEngine`, it had to be externalized. Rather then moving stuff around I just created a copy of that enum.